### PR TITLE
Release/2.0.0 and Doc Changes

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Descriptions:
1. MITRE Admin Role from all three dashboard.
2. Removed MITRE Widget from "Asset Risk Overview" dashboard.
3. Removed playbook "> Create Alerts Record"
4. Modified playbook "Scenario - OT - Create Alerts Record"
5. Changed File for "OT - Sample Assets CSV" and "OT - Sample Alerts CSV"
6. Changed min version compatibility.
7. Changed Asset Change Activity Scenario Data

### CODE REVIEW

File: docs/contents.md [hunk 1]
```diff
@@ -1,136 +1,151 @@
...
 
-### 00 - Use Case - Asset Change Activity
-
-|Playbook Name|Description|
-| :- | :- |
+### 08 - Utility - Asset Resources Management<sup>New<sup>
+
+| Playbook Name                                               | Description                                                                                                                          |
+| :---------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------- |
+| Set VOIP Phone's Asset Icon<sup>New<sup>                    | Playbook attach the Workstation Icon from the Asset Resources to the Asset for the VOIP Phone                                        |
```
Comment (cs-suyog-jinturkar) [inline docs/contents.md:96]: Playbook attach the Workstation Icon from the Asset Resources to the Asset for the VOIP Phone
It co-relates the VOIP Phone Asset with workstation Icon from the Asset Resources

File: playbooks/02 - Use Case - Asset Management/Scenario - OT - Create Assets Record.json [hunk 2]
```diff
@@ -170,35 +117,25 @@
...
             "isExecuted": false,
             "group": null,
-            "uuid": "76fdcbfb-a88b-4a96-a19c-f37cb9812f36"
+            "uuid": "5e6742ee-e393-4bd2-b1d0-c824b8f5a72a"
         },
         {
             "@type": "WorkflowRoute",
-            "name": "Start -> Configuration",
-            "targetStep": "\/api\/3\/workflow_steps\/85decb6e-d654-4141-8899-9febef97c45d",
+            "name": "Start -> asd",
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - Asset Management/Scenario - OT - Create Assets Record.json]: what is asd?

File: docs/usage.md [hunk 3]
```diff
@@ -25,17 +25,17 @@ The simulation mode has some sample data that helps you get a better understandi
 
     - Browse to `Simulations` > `OT - Add Sample Assets` scenario and click **Simulate Scenario**.
 
-    - This scenario adds 83 different sample OT Assets (different Levels, Types etc.) for testing dashboards, use case playbooks, reports and other actions.<br>
+    - This scenario adds 86 different sample OT Assets (different Levels, Types etc.) for testing dashboards, use case playbooks, reports and other actions.<br>
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: 85 or 86?

File: records/scenario/scenario0001.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
...
+        "description": "This scenario adds 85 different sample OT Assets (different Purdue Levels, Types etc.) for testing dashboards, use case playbooks, reports and other actions. These assets contain, a specific tag called 'Sample' so that they can easily be identified and if required, easily deleted later. The sample data is picked from a CSV file present in the 'OT - Sample Assets CSV' record in the 'Attachments' module - making it easier to modify the sample data as suitable. \n<br>\n\n**NOTE**: *Ensure you run this 'OT - Add Sample Assets' Scenario prior to running the 'OT - Add Sample Alerts', for facilitating the correlations of Alerts and Asset records.*",
```
Comment (cs-suyog-jinturkar) [inline records/scenario/scenario0001.json:6]: This scenario adds 86 different sample OT Assets (different Levels, Types etc.) for testing dashboards, use case playbooks, reports and other actions.<br>


85 or 86?

File: docs/usage.md (hunk 3 above)
Comment (cs-ankit-gupta) [reply docs/usage.md]: 85

File: records/scenario/scenario0001.json (hunk 4 above)
Comment (cs-ankit-gupta) [reply records/scenario/scenario0001.json:6]: 85

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**